dev/benchmarks/macrobenchmarks/lib/src/web/gallery/gallery_recorder.dart

```diff
@@ -0,0 +1,62 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
2020. Check all new files.

Reply:
This is fine. We like putting old years on everything. Makes us look retro :)